composer.json

```diff
@@ -25,6 +25,7 @@
         "nikic/php-parser": "^3.1.5",
         "openeuropa/behat-transformation-context" : "~0.1",
         "openeuropa/code-review": "~1.0@beta",
+        "openeuropa/oe_multilingual": "dev-master",
```

Comment:
Since we already have a release, let's use that one.

Reply:
It's require-dev, why?